docs/README.md

```diff
@@ -1,6 +1,40 @@
 # auth-service/Docs
 
 Documentation for the auth service
+## Database
+
+The following Figure depicts the Entity Relationship Diagram of the auth-service's schema:
+
+<p align="center">
+  <img src="http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/spikerheado1234/auth-service/master/docs/ErDiagram.puml">
+</p>
+
+The following table depicts the types of each attribute in the `Consent` table along with the constraints:
+
+|Attribute| Represented Internal Type | Constraints| Description |
+|:---------:|:---------------------------:|:------------:|:---------------------------:|
+|`id`|`VARCHAR(36)`|Unique - Primary Key|Alphanumeric consent ID|
+|`initiator_id`|`VARCHAR(32)`|Non-Null|Alphanumeric PISP ID|
+|`participant_id`|`VARCHAR(32)`|Non-Null|Alphanumeric DFSP ID|
+|`credential_id`|`INT UNSIGNED`|Unique - Non-Null|Unsigned numeric ID - upto 2^32 records|
+|`credential_type`|`VARCHAR(16)`|Non-Null|Alphanumeric enum value|
+|`credential_status`|`VARCHAR(10)`|Non-Null|String - `PENDING`/`ACTIVE`|
+|`credential_payload`|`TEXT`|Can be Null|Public key string|
+|`credential_challenge`|`CHAR(128)`|Non-Null|Base 64 encoded challenge string|
```

Comment:
A _Non-Null_ constraint here implies that the `credential_challenge` is created when the `Consent` object is created. 

However, in [1.5-grant-consent](https://github.com/jgeewax/pisp/blob/master/docs/linking/README.md#15-grant-consent), the credential object is still null, and the challenge isn't created until `CRED-5` in [1.6.1 Requesting a challenge](https://github.com/jgeewax/pisp/blob/master/docs/linking/README.md#161-requesting-a-challenge)